ssl/statem/extensions_clnt.c

```diff
@@ -16,10 +16,33 @@ EXT_RETURN tls_construct_ctos_renegotiate(SSL_CONNECTION *s, WPACKET *pkt,
                                           unsigned int context, X509 *x,
                                           size_t chainidx)
 {
-    /* Add RI if renegotiating */
-    if (!s->renegotiate)
-        return EXT_RETURN_NOT_SENT;
+    if (!s->renegotiate) {
+        /* If not renegotiating, send an empty RI extension to indicate support */
+
+        if (!SSL_CONNECTION_IS_DTLS(s)
+            && (s->min_proto_version >= TLS1_3_VERSION
+                || (ssl_security(s, SSL_SECOP_VERSION, 0, TLS1_VERSION, NULL)
+                    && s->min_proto_version <= TLS1_VERSION))) {
+            /*
+             * For TLS <= 1.0 SCSV is used instead, and for TLS 1.3 this
+             * extension isn't used at all.
```

Comment:
@fwh-dc - I guess this PR will mean some adaption will be required for the DTLSv1.3 work

Reply:
It would be very helpful if we could add some compile time check on 
`DTLS_MAX_VERSION == DTLS1_2_VERSION` to help spot this change when we rebase the `feature/dtls-1.3` branch.

Reply:
:+1: done